### DISCUSSION TITLE
How to use different certificate for different database (different urls) on the same machine?

### DISCUSSION THREAD

**Discussion by thunguyetng** [ravendb/ravendb#15412] [General]
I am learning RavenDB, I have a test database (database1), where I can access with a certificate (certification1) and it works well.

I need to access another database (database2) from another app, which has it own certificate. When I launch database2 on the browser, Chrome gives me error NET::ERR_CERT_INVALID.  MS Edge wants me to choose certification1. I cannot navigate to the path of certification2.

I registered for a new license. I got the email instructed me to register for my license. It seems that I can only replace certification1 with this new certificate (license). 

What can I do so I can access database2?

**Comment by haludi** [comment]
Hi,

I do not follow your question but if I do I think you mix between server
and client certificates.
When you create a server certificate with RavenDB studio (RavenDB uses
Let's Encrypt) the domain is bounded to the license.
If you want to reuse this domain you must use the same RavenDB license.
For the client certificate, you need to add/create it on the certificate
page -
https://ravendb.net/docs/article-page/5.4/csharp/server/security/authentication/certificate-management

Does it answer your question?

Best regards,
Igal

**Reply by thunguyetng** [reply]
Thank you for writing and for the link to certificate management.
I think this is server certificate.
I applied for a license, and I use it to access my test database1 from url1. This domain only has 1 database for now.
I applied for another license but I can not active it. Inside RavenDB Studio, I can only replace or renew license (activate and it will replace my current one).
So, I find the certificate file for domain2 (database2, url2), I installed the certificate on my machine using mmc app. I still have error in Chrome but MS Edge lets me choose the certificate and I can run RavenDB and access database2. I supposed the license to use domain2 is activated, I just need to install the certificate on my machine.
I hope I understand correctly. And may be one day I can run it on Chrome. Thank you again.

**Reply by ayende** [reply]
Licenses != certificates, mind.


There seems to be an issue with caching on your chrome. Try using an incognito browser and see what it does.

**Reply by thunguyetng** [reply]
@ayende & @haludi, thank you for your replies. I think I am still confused between licenses, client certificate and server certificate. I cleared my Chrome cache but not luck. I also used the incognito browser on Chrome, no luck either. MS Edge (not private browser) works for me. For now, I have what I need to continue learning RavenDB. Thank you again.

**Reply by garayx** [reply]
Hi @thunguyetng 
There might be an issue with Chrome/Edge with recent updates:
https://chromium.googlesource.com/chromium/src/+/main/net/data/ssl/chrome_root_store/faq.md

It might be that the new Chrome/Edge is using its own certificate store and not the host computer store.

Please try this fix by updating the registry:
https://www.atalasoft.com/kb2/KB/50418/FAQ-Changes-to-Google-Root-Store-Causes-WebCapture-WingScan-to-Stop-Working-Chrome-105-and-Later

**Reply by thunguyetng** [reply]
Hi @garayx , thank you for your post, it helps me to understand the root store and shows me how to change the setting in the registry not to use root store. It works for me. I can launch RavenDB Studio on Chrome. Thank you again.
